gatsby-node.js

```diff
@@ -327,3 +337,72 @@ exports.onCreatePage = async ({ page, actions }) => {
     createPage(page);
   }
 };
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions;
+  const typeDefs = `
+    type GradeCount {
+      grade: String
+      count: Int
+    }
+
+    type TypeCount {
+      type: String
+      count: Int
+    }
+
+    
+  `;
+  createTypes(typeDefs);
+};
+
+exports.createResolvers = ({ createResolvers }) => {
```

Comment:
I hope you haven't spent a lot of time on this.  The new GraphQL API is supposed to replace most of complicated graphql code in Gatsby

Reply:
Nah, plus I wanted to play around with graphQL queries.

Reply:
Plus I can keep moving forward building UI using that data, then I can figure out how to do it in the new API when you're ready